### PR TITLE
DHIS2-10981_FeedbackWidget

### PR DESCRIPTION
**Pull request by eirikhaugstulen**
Added documentation on the Feedback widget in the enrollment overview.

### CODE REVIEW

**Review by karolinelien — APPROVED** [review]
LGTM 👍

**Review by j-nguyenn — APPROVED** [review]

File: src/user/using-the-capture-app.md [hunk 1]
```diff
@@ -872,6 +872,17 @@ On the enrollment dashboard, you can view the Person Profile widget.
...
 
+### Feedback widget
+On the enrollment dashboard, you can view the Feedback widget.
+
+![](resources/images/capture_app/enrollment-dash-feedback-widget-1.png)
+
+The feedback widget displays text and key/value-pairs that are triggered by program rules and are not specific to any data element or attribute. 
+
+#### Empty state
+If there are no feedback for the specific context, the widget will display a text to inform that there are no feedbacks.
```
Comment (cooper-joe) [inline src/user/using-the-capture-app.md]: I suggest simplifying this sentence to:
`If there isn't any feedback for the current dashboard, the widget shows a short _empty_ message.`

File: src/user/using-the-capture-app.md [hunk 2]
```diff
@@ -872,6 +872,17 @@ On the enrollment dashboard, you can view the Person Profile widget.
...
+If there are no program rules that could potentially lead to a displayed feedback, the widget will be removed completely.
```
Comment (cooper-joe) [inline src/user/using-the-capture-app.md]: I suggest simplifying to:
`If there aren't any program rules that could show feedback for the current dashboard then the widget is hidden.`

**Review by cooper-joe — CHANGES_REQUESTED** [review]

File: src/user/using-the-capture-app.md [hunk 3]
```diff
@@ -872,6 +872,17 @@ On the enrollment dashboard, you can view the Person Profile widget.
 
 The Person profile widget is used for viewing key attributes but not for editing. In order to edit the person profile, you must select the `Edit` button to open an edit window. 
 
+### Feedback widget
+On the enrollment dashboard, you can view the Feedback widget.
```
Comment (cooper-joe) [inline src/user/using-the-capture-app.md]: I suggest removing this line and adding into the sentence below, see suggestion.

File: src/user/using-the-capture-app.md [hunk 4]
```diff
@@ -872,6 +872,17 @@ On the enrollment dashboard, you can view the Person Profile widget.
...
+
+![](resources/images/capture_app/enrollment-dash-feedback-widget-1.png)
+
+The feedback widget displays text and key/value-pairs that are triggered by program rules and are not specific to any data element or attribute.
```
Comment (cooper-joe) [inline src/user/using-the-capture-app.md]: I suggest simplifying and removing references to the DHIS2 data model:

`On the enrollment dashboard, the feedback widget displays text and values that are triggered by certain conditions. If the current dashboard triggers some rules set up in the program, the text or values will be automatically displayed.`

**Review by cooper-joe — CHANGES_REQUESTED** [review]

**Comment by eirikhaugstulen** [comment]
Thanks for the feedback! 
Implemented the changes 👍

**Review by cooper-joe — APPROVED** [review]

**Review by simonadomnisoru — APPROVED** [review]

**Pull request merged**